src/main/java/ch/wisv/events/ChConnectSecurityConfiguration.java

```diff
@@ -137,7 +146,9 @@ public AuthenticationProvider oidcAuthenticationProvider() {
                 CHUserInfo info = (CHUserInfo) userInfo;
 
                 if (properties.getAdminGroups().stream().anyMatch(info.getLdapGroups()::contains)) {
-                    return ImmutableSet.of(ROLE_ADMIN, ROLE_USER);
+                    return ImmutableSet.of(ROLE_ADMIN, ROLE_USER, ROLE_COMMITTEE);
+                } else if (info.getLdapGroups().size() > 0) {
```

Comment:
Deze check is misschien iets te kort door te bocht, omdat iedereen met een LDAP account in de`users` groep zit.

Hoe gebeurt de afscherming van het `/sales/` endpoint nu eigenlijk?

Reply:
Er is geen afscherming van de endpoint op dit moment, alleen je krijgt niks te zien